packages/run-protocol/src/econ-behaviors.js

```diff
@@ -330,25 +329,21 @@ export const startVaultFactory = async (
   instance.produce.VaultFactory.resolve(vaultFactoryInstance);
   instance.produce.Treasury.resolve(vaultFactoryInstance);
   instance.produce.VaultFactoryGovernor.resolve(governorInstance);
-  entries(installations).forEach(([name, install]) =>
-    installation.produce[name].resolve(install),
-  );
 };
 
 /**
  * Grant access to the VaultFactory creatorFacet
  * to up to one user based on address.
  *
- * @param { EconomyBootstrapPowers & {
- *   vatParameters: { argv: { vaultFactoryControllerAddress?: string } }
- * }} powers
+ * @param { EconomyBootstrapPowers } powers
+ * @param {object} [root0]
+ * @param {object} [root0.options]
+ * @param {string} [root0.options.vaultFactoryControllerAddress]
```

Comment:
`root0` looks like noise. consider `args` or `config`

Reply:
yes, some robot picked that. good catch. unfortunately we rushed a bit an missed this comment. Here's hoping to clean it up in due course...